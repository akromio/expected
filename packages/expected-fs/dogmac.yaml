uses:
  assert:
    js: assert
  expected:
    js: "@akromio/expected"
  fs:
    js: "@dogmalang/fs.sync"
  helpers:
    js: "@akromio/expected-helpers"
  path:
    js: "@dogmalang/path"