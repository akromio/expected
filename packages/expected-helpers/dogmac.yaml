uses:
  get:
    js: lodash.get
