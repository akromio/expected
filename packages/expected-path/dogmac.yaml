uses:
  assert:
    js: assert
  expected:
    js: "@akromio/expected"
  helpers:
    js: "@akromio/expected-helpers"
