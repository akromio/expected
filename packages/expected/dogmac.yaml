uses:
  get:
    js: lodash.get
  path:
    js: "@dogmalang/path"
  helpers:
    js: "@akromio/expected-helpers"
